packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -242,14 +242,6 @@ sealed class _DebugSemanticsRoleChecks {
     return error;
   }
 
-  static FlutterError? _semanticsMenu(SemanticsNode node) {
-    if (node.childrenCount < 1) {
-      return FlutterError('a menu cannot be empty');
```

Comment:
I think a `menu` can be empty if we have filter behavior, so I removed this check.